cloudsql/README.md

```diff
@@ -0,0 +1,56 @@
+## CloudSQL Example
+
+This example shows how to build an application that uses [Google Cloud SQL](https://cloud.google.com/sql/docs/introduction)
+using Kubernetes and [Docker](https://www.docker.com/).
+
+The example consists of a single pod containing two containers:
+
+- A web frontend container running Wordpress.
+- A [Cloud SQL Proxy](https://github.com/GoogleCloudPlatform/cloudsql-proxy/) container providing connectivity to Cloud SQL.
+
+### Prerequisites
+
+This example requires a running Kubernetes cluster at version 1.2 or higher. See the [Getting Started guides](../../docs/getting-started-guides/)
+for how to get started. And follow the [Prerequisites](../../docs/user-guide/prereqs.md) to make sure your `kubectl` is ok. As noted above, if
+you have a Google Container Engine cluster set up, go [here](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead.
+
+To check your version, run `kubectl version` and make sure both the client and server report a version of at least 1.2.
+
+You'll also need a running Cloud SQL instance, and a Service Account that can access the instance.
+See the [Cloud SQL getting started](https://cloud.google.com/sql/docs/access-control) page for more information.
+
+### Quickstart
+
+This section shows the simplest way to get the example work.
+
+#### Create Secrets
+
+You'll need to create several `Secret` resources to allow the SQL proxy to connect with your SQL instance. First, you'll need to
+create a secret resource containing the Service Account credentials to allow the proxy to communicate with the Cloud SQL API.
+
+First, download the JSON Service Account credentials.
+Then run this command, making sure to replace PATH_TO_CREDENTIAL_FILE with the correct location of the JSON file:
+
+```
+kubectl create secret generic cloudsql-oauth-credentials --from-file=credentials.json=<PATH_TO_CREDENTIAL_FILE>
+```
+
+Next, you'll need to create another pair of secrets to allow the proxy to connect to the actual SQL instance. This will contain
+the SQL username and password you'd like to connect as. Make sure to replace the USERNAME and PASSWORD values.
+
+```
+kubectl create secret generic cloudsql --from-literal=username=<USERNAME> --from-literal=password=<PASSWORD>
+```
+
+#### Create Pod
+
+Next, open the cloudsqlpod.yaml file in this repository and replace the values `$PROJECT`, `REGION` and `$INSTANCE`
+with the correct values for your SQL instance.
+
+Then, run:
+
+```
+kubectl create -f cloudsqlpod.yaml
+```
+
+to bring up the pod.
```

Comment:
It would be good to have some instruction on what to do next. How to see if it worked.